### PR TITLE
Update Pull Request template

### PR DESCRIPTION
**Pull request by pchasle**
**Description (for Contributor and Core Developer)**

Proposal for a simplified PR template

**Definition Of Done (for Core Developer only)**

- [x] Tests
- [x] Migration & Installer
- [x] PM Validation (Story)
- [x] Changelog (maintenance bug fixes)
- [x] Tech Doc

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
+<!---
+<3 Thanks for taking the time to contribute! You're awesome! <3
+
+If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md
+
+Checklist (for Core Developer only):
+- Tests
+- Migration & Installer
+- PM Validation (Story)
+- Changelog (Bug Fix)
```
Comment (ahocquard) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: not only bug fix :thinking: it is also BC breaks (and mainly on master)

**Review by ValentinMumble — APPROVED** [review]

**Review by ahocquard — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (pchasle) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: :+1: I will add that `Changelog (Bug Fix & BC breaks)`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,25 +1,15 @@
...
-| Migration script                  | -
-| Tech Doc                          | -
-
-`Todo`: Pending / Work in progress
-`OK`: Done / Validated
-`-`: Not needed
+- [ ] Tests (avoid end-to-end)
+- [ ] Migration & Installer
+- [ ] PM Validation (Story)
+- [ ] Changelog (Bug Fix & BC break)
```
Comment (StevenVAIDIE) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: Now we didn't add BC break line in Changelog in master (cf: https://akeneo.slack.com/archives/G033H0MJ2/p1612450492062800)
```suggestion
- [ ] Changelog (Bug Fix)
```

**Review by StevenVAIDIE — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,25 +1,15 @@
...
-| Changelog updated                 | Todo
-| Review and 2 GTM                  | Todo
-| Micro Demo to the PO (Story only) | Todo
-| Migration script                  | -
-| Tech Doc                          | -
-
-`Todo`: Pending / Work in progress
-`OK`: Done / Validated
-`-`: Not needed
+- [ ] Tests (avoid end-to-end)
```
Comment (BitOne) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
- [ ] Tests
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (BitOne) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
- [ ] Changelog (maintenance bug fixes)
```

**Review by BitOne — APPROVED** [review]
GTM after fix.

**Review by willy-ahva — APPROVED** [review]

**Review by pierrejolly — APPROVED** [review]

**Pull request merged**